src/averageAges.js

```diff
@@ -15,11 +19,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const malePeople = people.filter(person =>
+    person.sex === MALE
+    && (century === undefined
+    || century === Math.ceil(person.died / CENTURY)));
+
+  const menAgesSum = malePeople.reduce((total, person) =>
+    total + (person.died - person.born), 0);
```

Comment:
Remember about "DRY". You have the same calculation in the second function. Create a separate function for this